src/ResourceManager/TrafficManager/Commands.TrafficManager2/ExpectedStatusCodeRange/RemoveAzureTrafficManagerExpectedStatusCodeRange.cs

```diff
@@ -0,0 +1,69 @@
+﻿//using System;
+//using System.Collections.Generic;
+//using System.Linq;
+//using System.Text;
+//using System.Threading.Tasks;
+
+//namespace Microsoft.Azure.Commands.TrafficManager.ExpectedStatusCodeRange
+//{
+//    class RemoveAzureTrafficManagerExpectedStatusCodeRange
+//    {
+//    }
+//}
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.TrafficManager.Models;
+using Microsoft.Azure.Commands.TrafficManager.Utilities;
+using System.Management.Automation;
+using ProjectResources = Microsoft.Azure.Commands.TrafficManager.Properties.Resources;
+
+namespace Microsoft.Azure.Commands.TrafficManager
+{
+    using System;
+
+    [Cmdlet(VerbsCommon.Remove, "AzureRmTrafficManagerExpectedStatusCodeRange"), OutputType(typeof(TrafficManagerProfile))]
+    public class RemoveAzureTrafficManagerExpectedStatusCodeRange : TrafficManagerBaseCmdlet
+    {
+        [Parameter(Mandatory = true, HelpMessage = "The minimum value of the expected status code range to be removed.", ParameterSetName = "Fields")]
+        [ValidateNotNullOrEmpty]
+        public int? Min { get; set; }
```

Comment:
Since the param is mandatory, it PS will make sure a  value is specified for the param, it can not be null.

Instead of marking the param as a nullable int, id leave it as an int and instead of [ValidateNotNullOrEmpty] maybe use [ValidateRange] is there is a certain range the values will be in.